src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -0,0 +1,137 @@
+    <attribute side="server" code="0x0029" define="THERMOSTAT_RUNNING_STATE" type="BITMAP16" writable="false" optional="true" introducedIn="ha-1.2-05-3520-29">hvac relay state</attribute>
+    <attribute side="server" code="0x0030" define="SETPOINT_CHANGE_SOURCE" type="ENUM8" writable="false" optional="true" introducedIn="ha-1.2-05-3520-29">setpoint change source</attribute>
+    <attribute side="server" code="0x0031" define="SETPOINT_CHANGE_AMOUNT" type="INT16S" writable="false" default="0x8000" optional="true" introducedIn="ha-1.2-05-3520-29">setpoint change amount</attribute>
+    <attribute side="server" code="0x0032" define="SETPOINT_CHANGE_SOURCE_TIMESTAMP" type="epoch_s" writable="false" optional="true" introducedIn="ha-1.2-05-3520-29">setpoint change source timestamp</attribute>
+    <attribute side="server" code="0x0040" define="AC_TYPE" type="ENUM8" min="0x00" max="0x04" writable="true" default="0x00" optional="true" introducedIn="ha-1.2-05-3520-29">ac type</attribute>
+    <attribute side="server" code="0x0041" define="AC_CAPACITY" type="INT16U" min="0x0000" max="0xFFFF" writable="true" default="0x0000" optional="true" introducedIn="ha-1.2-05-3520-29">ac capacity</attribute>
+    <attribute side="server" code="0x0042" define="AC_REFRIGERANT_TYPE" type="ENUM8" min="0x00" max="0x03" writable="true" default="0x00" optional="true" introducedIn="ha-1.2-05-3520-29">ac refrigerant type</attribute>
+    <attribute side="server" code="0x0043" define="AC_COMPRESSOR" type="ENUM8" min="0x00" max="0x03" writable="true" default="0x00" optional="true" introducedIn="ha-1.2-05-3520-29">ac compressor</attribute>
+    <attribute side="server" code="0x0044" define="AC_ERROR_CODE" type="BITMAP32" min="0x00000000" max="0xFFFFFFFF" writable="true" default="0x00000000" optional="true" introducedIn="ha-1.2-05-3520-29">ac error code</attribute>
+    <attribute side="server" code="0x0045" define="AC_LOUVER_POSITION" type="ENUM8" min="0x00" max="0x05" writable="true" default="0x00" optional="true" introducedIn="ha-1.2-05-3520-29">ac louver position</attribute>
+    <attribute side="server" code="0x0046" define="AC_COIL_TEMPERATURE" type="INT16S" min="0x954D" max="0x7FFF" writable="false" default="0x8000" optional="true" introducedIn="ha-1.2-05-3520-29">ac coil temperature</attribute>
+    <attribute side="server" code="0x0047" define="AC_CAPACITY_FORMAT" type="ENUM8" min="0x00" max="0xFF" writable="true" default="0x00" optional="true" introducedIn="ha-1.2-05-3520-29">ac capacity format</attribute>
+    <command source="client" code="0x00" name="SetpointRaiseLower" optional="false" cli="zcl tstat set">
+      <description>
+        Command description for SetpointRaiseLower
+      </description>
+      <arg name="mode" type="SetpointAdjustMode"/>
+      <arg name="amount" type="INT8S"/>
+    </command>
+    <command source="client" code="0x01" name="SetWeeklySchedule" optional="false" cli="zcl tstat sws" introducedIn="ha-1.2-05-3520-29">
```

Comment:
Scheduling commands are optional per the spec.  They would be required if Feature Map has Schedule flag set. 
So I think strictly speaking scheduling commands are optional.

Reply:
Well, I have not looked in to the detail content since it is the same one as in ha.xml. All I did is move it. If there are generic comments about the content of the XML or if something is not according to the spec someone responsible for it is welcome to update it (even in this PR).

The main work I did here is adding the feature map.